src/pages/study-room/components/participant-list.test.tsx

```diff
@@ -0,0 +1,58 @@
+import { fireEvent, screen } from '@testing-library/react';
+import { render } from '@/test-helper';
+import fixtures from '@/fixtures';
+import { UsePopover } from '../hooks/usePopover';
+import ParticipantList from './participant-list';
+
+const context = describe;
+
+const { participants } = fixtures;
+
+const mockUsePopover: UsePopover = {
+  anchorEl: null,
+  onOpen: jest.fn(),
+  onClose: jest.fn(),
+};
+
+jest.mock('../hooks/usePopover', () => () => mockUsePopover);
+
+const renderParticipantList = () => render(<ParticipantList participants={participants} />);
+
+describe('ParticipantList', () => {
+  describe('when anchorEl is null', () => {
+    it('renders only participant list button', () => {
+      renderParticipantList();
+
+      screen.getByText(participants.length);
+
+      expect(screen.queryByText(participants[0].nickname)).not.toBeInTheDocument();
+    });
+
+    context('when click participant list button', () => {
+      it('renders participant list', () => {
+        renderParticipantList();
+
+        const buttonNameRegExp = new RegExp(`${participants.length}`);
```

Comment:
number -> string 형변환을 위해서라면 이렇게는 어떨까요?
```suggestion
        const buttonNameRegExp = new RegExp(String(participants.length));
```